hw3/hw3-v1tal4ik/src/people.data.js

```diff
@@ -0,0 +1,48 @@
+export const SarahMay = {
```

Comment:
why did you copy this file to this folder? I suppose that it should work if you import it from the root folder

Reply:
yes. it's works, but I use babel and he watching src folder and compiling to dist :)